metrics.py

```diff
@@ -58,6 +58,7 @@ def get(self):
 
       memcache.set(KEY, data, time=CACHE_AGE)
 
+    data = self._clean_data(data)
```

Comment:
Should the JSON handler just call `self._clean_data(data)` instead of dropping the calls in here and below?